kubernetes-model-generator/kubernetes-model-core/src/main/java/io/fabric8/kubernetes/api/model/HasMetadata.java

```diff
@@ -493,4 +495,55 @@ default void removeOwnerReference(HasMetadata owner) {
   default Optional<ObjectMeta> optionalMetadata() {
     return Optional.ofNullable(getMetadata());
   }
+
+  /**
+   * Initializes this {@link ObjectMeta} field with name and namespace (if this instance represents a namespaced resource)
+   * provided by the specified HasMetadata instance. This is a convenience method to avoid boilerplate, notably when using
+   * Server-Side Apply, when creating a new instance with only some fields of the original one. Calls
+   * {@link #setMetadata(ObjectMeta)} when done, if you want to further configure this instance's metadata, please use
+   * {@link #initMetadataBuilderNameAndNamespaceFrom(ObjectMeta)} instead, which <em>doesn't</em> sets the metadata, leaving it
+   * up to the user once configuration is finished.
+   *
+   * @param original a HasMetadata instance from which to retrieve the name and namespace
+   */
+  default void initNameAndNamespaceFrom(HasMetadata original) {
```

Comment:
Can we omit this signature? It duplicates a lot of javadoc, but add's only a minor amount of convenience.

Reply:
I'd rather keep this one than the `ObjectMeta` one, actually since, when using SSA, that's what you'll most likely start with (a resource).